cloudferrylib/os/actions/check_networks.py

```diff
@@ -87,8 +86,90 @@ def run(self, **kwargs):
             LOG.critical('Network overlapping list:\n%s', overlapped_resources)
             raise exception.AbortMigrationError(
                 "There is a number of overlapping Network resources, so "
-                "migration process can not be continued. Resolve it please and"
+                "migration process can not "
+                "be continued. Resolve it please and"
                 " try again.")
+        self.check_conflicts_ports(src_net,
+                                   dst_net)
+
+    def check_conflicts_ports(self, src_net, dst_net):
+        LOG.info("Detecting ports conflicts...")
+        src_ports = self.get_ports('compute:', src_net)
+        dst_ports = self.get_ports('compute:', dst_net)
+        src_ports_by_fixip = self.ports_by_fixed_ips(src_ports)
+        dst_ports_by_fixip = self.ports_by_fixed_ips(dst_ports)
+        src_ports_by_mac = self.ports_by_mac(src_ports)
+        dst_ports_by_mac = self.ports_by_mac(dst_ports)
+        self.check_intersects_ports_fixips(src_ports_by_fixip,
+                                           dst_ports_by_fixip)
+        self.check_intersects_ports_mac(src_ports_by_mac,
+                                        dst_ports_by_mac)
+
+    def check_intersects_ports_mac(self,
+                                   src_ports_by_mac,
+                                   dst_ports_by_mac):
+        src_ports = set(src_ports_by_mac)
+        dst_ports = set(dst_ports_by_mac)
+        inter_mac = src_ports & dst_ports
+        for mac in inter_mac:
+            LOG.warning("---------------")
+            LOG.warning("Conflicts ports by mac...")
+            LOG.warning("SRC port: port id: %s" %
+                        src_ports_by_mac[mac]['id'])
+            LOG.warning("          mac: %s" % mac)
+            LOG.warning("DST port: port id: %s" %
+                        dst_ports_by_mac[mac]['id'])
+            LOG.warning("          mac: %s" % mac)
+            LOG.warning("---------------")
+        if inter_mac:
+            raise exception.AbortMigrationError("Src and dst "
+                                                "clouds have conflicts "
+                                                "ports by mac")
```

Comment:
Current warning messages would be hardly useful to CF operator, because they don't provide enough information. Since there's an overlap, CF operator would probably need following information to resolve the problem manually:
 - Network name and ID
 - Port name (if available) and ID
 - MAC address
Please update all messages to include that information.